corehq/apps/hqwebapp/static/hqwebapp/js/alpinejs/directives/datepicker.js

```diff
@@ -76,10 +76,21 @@ Alpine.directive('datepicker', (el, { expression }, { cleanup }) => {
             theme: 'light',
             components: components,
             icons: faFiveIcons,
+            buttons: {
```

Comment:
This commit would also be really nice to share with web apps by doing something like pulling the config into a file that could be shared.

Reply:
great idea, will do